### ISSUE TITLE
Redis 6.2.7-debian-11-r18 fails to start in TLS mode without a passphrase file

### ISSUE THREAD

**Issue by jonseitz** [bitnami/containers#1602]
### Name and Version

bitnami/redis:6.2.7-debian-11-r18

### What steps will reproduce the bug?

1. Run redis/bitnami:6.2.7-debian-11-r18 with [this script][1] mounted as the as the entrypoint.

[1]: https://github.com/seas-computing/course-planner/blob/f8754bfd708c3468c3d6ff702224952561c41736/init-redis-ssl.sh

### What is the expected behavior?

I'm running the `bitnami/redis` container in a development-only local environment, where I need to simulate running Redis in TLS mode by generating a self-signed certificate when the container is initialized. Since this is not a secure environment, the key doesn't need to be encrypted.

Prior to `6.2.7-debian-11-r18`, the redis container would start up normally and run in TLS mode. 

![2022-08-02_122256](https://user-images.githubusercontent.com/6124793/182424826-eedd5aa0-675f-4d68-b159-05862ce04333.png)

### What do you see instead?

Starting with version `6.2.7-debian-11-r18`, it prints the following error and crashes:
 
```
"The passphrase for the private key file in the specified path  does not exist"
```

![2022-08-02_122329](https://user-images.githubusercontent.com/6124793/182424887-f88741b8-ca46-4bb2-ba7e-a750330810c8.png)

### Additional information

I've currently worked around this by creating an empty file in the init script and using that as the passphrase variable. This passes the `! -f "$REDIS_TLS_KEY_FILE_PASS"` check added to [`libredis.sh`][2] in #359. Perhaps this check could be changed to only check for the existence of the `KEY_FILE_PASS` file if that environment variable is actually defined. Happy to open a PR to that end if this would be acceptable.

[2]:https://github.com/bitnami/containers/pull/359/files#diff-ebedba9f582a099041caa636ecd3b4d8df4437c1f95f059b02ef5d737188730dR223-R225

**Comment by migruiz4** [comment]
Hi @jonseitz,

Thank you very much for reporting this issue! 

I'm working on a fix for this issue, I will let you know once a new version of the `bitnami/redis` image is released.

**Comment by migruiz4** [comment]
Hi again @jonseitz,

We just released the tags `6.2.7-debian-11-r25` and `7.0.4-debian-11-r11` of the `bitnami/redis` image including a fix for this issue.

Could you please let me know if it solves the issue?

**Comment by migruiz4** [comment]
Thank you for your feedback! In that case, I will now close this issue
